<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardView</class>
 <widget class="QWidget" name="DashboardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   
   <item>
    <layout class="QVBoxLayout" name="leftPanel">
     <item>
      <widget class="QGroupBox" name="groupInfo">
       <property name="title">
        <string>Witaj!</string>
       </property>
       <layout class="QVBoxLayout" name="vboxInfo">
        <item>
         <widget class="QLabel" name="lblWelcome">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Użytkownik</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCurrentDate">
          <property name="styleSheet">
           <string notr="true">color: gray;</string>
          </property>
          <property name="text">
           <string>Data</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>

     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #232336; color: white; selection-background-color: #4CAF50;</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QGroupBox" name="groupStats">
       <property name="title">
        <string>Szybkie Statystyki</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="l1"><property name="text"><string>Treningi w tym miesiącu:</string></property></widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblMonthCount">
          <property name="text"><string>-</string></property>
          <property name="styleSheet"><string notr="true">font-weight: bold; color: #4CAF50;</string></property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     
     <item>
      <spacer name="verticalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QVBoxLayout" name="rightPanel">
     <item>
      <widget class="QGroupBox" name="groupFilters">
       <property name="title">
        <string>Analiza Progresu</string>
       </property>
       <layout class="QHBoxLayout" name="filtersLayout">
        <item>
         <widget class="QLabel" name="lEx"><property name="text"><string>Ćwiczenie:</string></property></widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboExercises">
          <property name="minimumSize"><size><width>150</width><height>0</height></size></property>
         </widget>
        </item>
        
        <item>
         <widget class="QLabel" name="lFrom"><property name="text"><string>Od:</string></property></widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateFrom">
          <property name="calendarPopup"><bool>true</bool></property>
          <property name="displayFormat"><string>yyyy-MM-dd</string></property>
         </widget>
        </item>
        
        <item>
         <widget class="QLabel" name="lTo"><property name="text"><string>Do:</string></property></widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateTo">
          <property name="calendarPopup"><bool>true</bool></property>
          <property name="displayFormat"><string>yyyy-MM-dd</string></property>
         </widget>
        </item>

        <item>
         <widget class="QPushButton" name="btnRefreshAnalysis">
          <property name="text">
           <string>Pokaż Wykres</string>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #5C6BC0; color: white; font-weight: bold;</string>
          </property>
         </widget>
        </item>
        
        <item>
         <spacer name="horizontalSpacerFilters">
          <property name="orientation"><enum>Qt::Horizontal</enum></property>
         </spacer>
        </item>

        <item>
         <widget class="QLabel" name="lblMax1RM">
          <property name="text"><string>Rekord (1RM): - kg</string></property>
          <property name="styleSheet"><string notr="true">color: #00E5FF; font-weight: bold; border: 1px solid #00E5FF; padding: 5px; border-radius: 4px;</string></property>
         </widget>
        </item>

       </layout>
      </widget>
     </item>

     <item>
      <widget class="QWidget" name="chartContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #232336; border: 1px solid #3E3E5E; border-radius: 8px;</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QTableWidget" name="tableHistory">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column><property name="text"><string>Data</string></property></column>
       <column><property name="text"><string>Ciężar</string></property></column>
       <column><property name="text"><string>Powtórzenia</string></property></column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>